src/scores.ts

```diff
@@ -37,10 +20,10 @@ async function calculateScores(parent, args, key) {
     parent.requestId
   );
 
-  let snapshotBlockNum = 'latest';
-  if (args.snapshot !== 'latest') {
```

Comment:
This code is working fine, any reason to change it? With longer code

Reply:
Hey yes,, trying to do the same thing as in `getVp` to make it work with `latest` if we pass any unknown parameter into the snapshot param